css/index.css

```diff
@@ -0,0 +1,90 @@
+/* Font Import */
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600&display=swap');
+
+/* Global Variables */
+:root {
+    --main-page-text-color: #363442;
+    --primary-blue: #0101fe;
+    --pure-white: #FFFFFF;
+    --white: #e8e8fd;
+    --black: #18181B;
+    --dark: #20232A;
+    --grey: #E5E7EB;
+}
+
+/* Default Styling */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Montserrat', sans-serif;
+}
+
+/* No Scrollbar */
+::-webkit-scrollbar {
+    display: none;
+}
+
+/* Full Page Card 1 */
+.full-page-card-one {
+    background-image: url(/img/bg-one_1280x800.jpg);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 100vh;  
```

Comment:
You can use short properties for background 
background: url(/img/bg-one_1280x800.jpg) no-repeat center cover;

Reply:
Yes, I changed it.
But the shorthand example you provided didn't work.

According to MDN and W3, there needs to be a slash separating the background-size from the background-position.
[Link regarding this](https://stackoverflow.com/a/17093346)

The correct syntax that worked is : 
`background: url(/img/bg-one_1280x800.jpg) center/cover no-repeat;`